src/lakefs_spec/spec.py

```diff
@@ -91,6 +91,38 @@ def parse(path: str) -> tuple[str, str, str]:
     return repo, ref, resource
 
 
+def ensure_branch(
+    client: LakeFSClient, repository: str, branch_name: str, source_branch_name: str = "main"
```

Comment:
Could you rename these arguments to `branch` and `source_branch`, respectively? Same in the docstrings.